content/api/events-ingest.apib

```diff
@@ -81,6 +81,40 @@ Required Headers
             }
         }
 
+### Get Ingest Events [GET /v1/ingest/events/{id}]
```

Comment:
This should be `Get Events` or we should change the POST title to `Send Ingest Events`